apps/src/templates/teacherDashboard/SectionRow.jsx

```diff
@@ -134,8 +134,18 @@ class SectionRow extends Component {
 
   onClickDeleteNo = () => this.setState({deleting: false});
 
-  // TODO(bjvanminnen)
-  onClickDeleteYes = () => console.log('this is where our delete will happen');
+  onClickDeleteYes = () => {
+    const { section, removeSection } = this.props;
+    $.ajax({
+      url: `/v2/sections/${section.id}`,
+      method: 'DELETE',
+    }).done(() => {
+      removeSection(section.id);
+    }).fail((jqXhr, status) => {
+      // TODO(bjvanminnen): figure out how what we want to do in this case
```

Comment:
Addressed in upcoming PR https://github.com/code-dot-org/code-dot-org/pull/15941